### ISSUE TITLE
Persona Coin isImageLoaded not reset after removing image

### ISSUE THREAD

**Issue by nico-bellante** [OfficeDev/office-ui-fabric-react#6570]
### Please provide a reproduction of the bug in a codepen:

https://codepen.io/Pengurino/pen/mzyJrO?editors=0010

(Click the Hello World button twice to simulate behavior.)

### Bug Report 
 
 
- __Package version(s)__: <script src="//unpkg.com/office-ui-fabric-react/dist/office-ui-fabric-react.min.js"></script> and 6.73.0

 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) If I can find time
 
Requested priority: (Blocking, High, Normal, Low) High
 
Products/sites affected: https://script-lab-react-alpha.azurewebsites.net
 
#### Describe the issue:
When the image is removed from the Persona Coin (set back to undefined), the initials do not re-appear.
 
#### Actual behavior:
When the image is removed from the Persona Coin (set back to undefined), the initials do not re-appear.
 
#### Expected behavior:
The initials should re-appear when the imageUrl is set to undefined.

**Comment by Vitalius1** [comment]
@nico-bellante thanks for submitting this issue and providing with a repro.
Looks like a simple change like this will fix the issue (Tested in codepen and indeed it works)
```
public componentWillReceiveProps(nextProps: IPersonaCoinProps): void {
    if (!nextProps.imageUrl) {
      this.setState({
        isImageLoaded: false
      });
    }
  }
```

**Comment by gregdardis** [comment]
Mind if I open a PR with this fix?

**Comment by Vitalius1** [comment]
> Mind if I open a PR with this fix?

Yeah. Also would be nice to have some kind of unit test checking for this.
